### ISSUE TITLE
Instalacja wersji z biblioteki python 2.7 zamiast 3.8? pytest

### ISSUE THREAD

**Issue by eleeri** [daftcode/daftacademy-python_levelup-spring2020#5]
Przy komendzie 'pytest' wyskakuje poniższy błąd, żeby zadziałało muszę użyć 'python3.8 -m pytest'. Wygląda na to, że werjsa pytest inastaluje się z biblioteki pythona 2.7, mimo wirtualnego środowiska i korzystania w nim z pythona 3.8. Nie wiem czy da się to jakoś zmienić/naprawić?

![obraz](https://user-images.githubusercontent.com/11413254/78299423-2fde4100-7536-11ea-9518-4be186ca6201.png)

**Comment by mateusz94** [comment]
czy instalowałeś pytest w tym środowisku wirtualnym?

**Comment by mateusz94** [comment]
wydaję mi się, że:
`python -m pytest`
też zadziała

**Comment by eleeri** [comment]
Tak, pytest był instalowany w środowisku wirtualnym, więc jest to dla mnie zagadkowe. 
i zgadza się, 
python -m pytest też działa. 
Nie działa tylko samo pytest

**Comment by mateusz94** [comment]
spróbuj zrobić taki manewr:
`deactivate && source deploy/bin/activate`

https://stackoverflow.com/questions/35045038/how-do-i-use-pytest-with-virtualenv

**Comment by mateusz94** [comment]
@eleeri i jak?

**Comment by eleeri** [comment]
działa, dziękuję :) Acz sublime przestał się odpalać dla odmiany, zawsze coś musi się wykrzaczać ;)
